arch/arm64/boot/dts/qcom/msm8953-xiaomi-vince.dts

```diff
@@ -17,6 +17,21 @@
 	compatible = "xiaomi,vince", "qcom,msm8953";
 	qcom,board-id= <0x1000b 0x08>;
 
+	gpio-keys {
+		compatible = "gpio-keys";
+
+		pinctrl-names = "default";
+		pinctrl-0 = <&gpio_key_default>, <&gpio_hall_sensor_default>;
```

Comment:
You're somewhat weirdly overwriting/extending the gpio-keys node from xiaomi-common.dtsi, so logically `gpio_key_default` doesn't fit here at all also.